posts/three-ways-to-search-video.md

```diff
@@ -2,8 +2,9 @@
 title: Three Ways to Search with Sourcegraph (Video)
 tags: [video, search, sourcegraph]
 author: marek-zaluski
-image: https://i3.ytimg.com/vi/XLfE2YuRwvw/maxresdefault.jpg
+image: https://learn.sourcegraph.com/headers/three-ways-to-search-header.jpg
```

Comment:
Does this one need to be converted?

Reply:
yes

Reply:
Actually, I think we don't need the image, just the social image? Not sure if we need a banner on this, @marekweb what do you think?